modules/shapes/module.php

```diff
@@ -15,11 +15,11 @@ class Module extends \Elementor\Core\Base\Module {
 	 */
 	public static function get_paths( $add_custom = true ) {
 		$paths = [
+			'wave' => __( 'Wave', 'elementor' ),
```

Comment:
a-z?

Reply:
The Product want this one to be first 🤷‍♂️